base/applications/cmdutils/where/lang/it-IT.rc

```diff
@@ -0,0 +1,61 @@
+/* 
+ * PROJECT:     ReactOS WHERE command
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Italian resource file
+ * TRANSLATOR:  Copyright 2023 Carlo Bramini
```

Comment:
Author email is missing; See new-style header example: https://github.com/reactos/reactos/commit/2f70f79904a480d800e23f4f47bb8d30a5da913b#diff-473b35ab24607c4ee17ac5ace2c0a53f6e5d0a23ac181994448a4d5d200e53f0